test/Npgsql.Tests/Types/JsonPathTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Threading.Tasks;
+using NpgsqlTypes;
+using NUnit.Framework;
+using static Npgsql.Tests.TestUtil;
+
+namespace Npgsql.Tests.Types
+{
+    public class JsonPathTests : MultiplexingTestBase
+    {
+        public JsonPathTests(MultiplexingMode multiplexingMode)
+            : base(multiplexingMode) { }
+
+        static readonly object[] ReadWriteCases = new[]
```

Comment:
Is there a need for us to test different cases here? After all Npgsql is just reading and writing strings here and doesn't interact with the content in any way... If that's the case, I'd consider just embedding any same query (one of the below) in the tests themselves for simplification

Reply:
An experiment, looking for unification of our type handlers tests, so in the final they should just accept cases and have test methods in the base class like in benchmarks. No boilerplate code, just cases.

Reply:
The difference is that in numeric there really are different cases for us to test, because they trigger different code paths inside the handler. With jsonpath we are just passing a string..

Reply:
Yes, but the idea is to have a common base for all type tests. Doesn't matter what is going to be tested, it tested the same way.

Reply:
I'm not sure I see the point of that. Am OK for merging as-is, but maybe better to do test cases when we actually apply the common base?

If you want to keep this, is anything additional being tested by having two test cases? Why not just keep one? I'm asking because the test gives the impression that we actually care what's in those strings.

Reply:
The next step is to start test unification, so the answer to your question is no (:

To see why I added the second case, just remove double quotes from the query string and see what happens. While `jsonpath` is transferred as a C string prefixed by a version number, the internal representation isn't a string and casting it back could give a different result.